broadcaster/_backends/postgres.py

```diff
@@ -25,7 +26,8 @@ async def unsubscribe(self, channel: str) -> None:
         await self._conn.remove_listener(channel, self._listener)
```

Comment:
Does this `self._conn.remove_listener` call (and the `self._conn.add_listener` above) need the lock too or just the `self._conn.execute`?

Reply:
indeed!